### ISSUE TITLE
Artifacts

### ISSUE THREAD

**Issue by Trentin996** [Azanor/thaumcraft-suggestions#57]
It would be cool to have possibly large, but very dangerous, dungeons of sorts housing a single powerful artifacts each. Maybe something like a ring that would decrease the Vis cost of Foci but also increase their power? Or a necklace already enchanted with an exceedingly high amount of Runic Shielding, maybe a Vis discount in addition to that. Just general things of that nature that would allow a Thaumaturge more power, but not without a huge amount of risk to life and limb. Possibly even add a few new bosses to guard them.

**Comment by Argis13** [comment]
Hmmm... maybe only openable with a drop from the outer lands? Cause these are only useful to Thaumaturges, and they sound pretty hard. (And add some more content to the eldritch tab _cough_ _cough_)

**Comment by Trentin996** [comment]
Possibly so, yes. Maybe create some form of object that doesn't break out of a Primordial Pearl to open the dungeons?

**Comment by Argis13** [comment]
I was thinking more a eldrich form of a creeper to drop an tainted gunpowder to blow up the door way.

**Comment by Trentin996** [comment]
That would be too simple - too easy. In making one visit the Outer Lands first and kill a least a single boss to try to get there would force one to be exceedingly powerful before they were even allowed entrance.

**Comment by Argis13** [comment]
True... How about the item is dropped from the pots in the boss room, but is consumed when you open a door. That way you don't have to spend any of those precious pearls, but you still have to beat the boss to enter the dungeon, and multiple bosses if you want all the items.

**Comment by SpiderOverlord** [comment]
in essence the crimson blade is an artifact
